oci/spec_test.go

```diff
@@ -223,6 +223,32 @@ func TestWithCapabilitiesNil(t *testing.T) {
 	}
 }
 
+func TestPopulateDefaultWindowsSpec(t *testing.T) {
+	var (
+		c   = containers.Container{ID: "TestWithDefaultSpec"}
+		ctx = namespaces.WithNamespace(context.Background(), "test")
+	)
+	var expected Spec
+
+	populateDefaultWindowsSpec(ctx, &expected, c.ID)
+	if expected.Windows == nil {
+		t.Error("Cannot populate windows Spec")
+	}
+}
+
+func TestPopulateDefaultUnixSpec(t *testing.T) {
+	var (
+		c   = containers.Container{ID: "TestWithDefaultSpec"}
+		ctx = namespaces.WithNamespace(context.Background(), "test")
+	)
+	var expected Spec
+
+	populateDefaultUnixSpec(ctx, &expected, c.ID)
+	if expected.Linux == nil {
```

Comment:
These kind of checks already covered here https://github.com/containerd/containerd/pull/3830/files#diff-06da64bd4db31eae8d2cd08acef7ec97R94-R110 
This explains why codecov report wasn't actually improved by this PR.
If you want to increase it, I'd rather suggest looking at `spec_opts.go` or `spec_opts_linux.go`, which has pretty [low coverage](https://codecov.io/gh/containerd/containerd/pull/3830/tree/oci)

Reply:
@mxpv got it, i will continue improving testing coverage of `spec_opts.go` and `spec_opts_linux.go` in this PR.